app/ui-angular/pom.xml

```
@@ -121,39 +124,47 @@
           </execution>
           <execution>
             <id>yarn ng lint</id>
+            <phase>compile</phase>
```

```suggestion
            <phase>verify</phase>
```

So not to trigger lint on `mvn compile`

---

Changed